src/modules/thesaurus/src/main/java/org/geonetwork/thesaurus/util/LanguageCodeConverter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * (c) 2003 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license,
+ * available at the root application directory.
+ */
+package org.geonetwork.thesaurus.util;
+
+import java.util.Locale;
+
+public class LanguageCodeConverter {
+
+    public static String toTwoLetterCode(String threeLetterCode) {
```

Comment:
This is a consistently annoying problem responsible for lots of confusion when learning GeoNetwork. A small javadoc here can help others.

```suggestion
    /**
     * Convert ISO three letter language code to Java two letter Locale name.
     * 
     * @param threeLetterCode ISO three letter language code
     * @return {@link Locale#getLanguage} two letter code, defaults to {@code en} if not matched.
     */
    public static String toTwoLetterCode(String threeLetterCode) {
```